pkgs/data/themes/qtcurve/default.nix

```
@@ -1,32 +1,24 @@
-{ stdenv, fetchFromGitHub, cmake, extra-cmake-modules, pkgconfig, mkDerivation
+{ lib, fetchFromGitHub, cmake, extra-cmake-modules, pkgconfig, mkDerivation
 , gtk2, qtbase, qtsvg, qtx11extras # Toolkit dependencies
 , karchive, kconfig, kconfigwidgets, kio, frameworkintegration
 , kguiaddons, ki18n, kwindowsystem, kdelibs4support, kiconthemes
 , libpthreadstubs, pcre, libXdmcp, libX11, libXau # X11 dependencies
 , fetchpatch
 }:
 
-let
-  version = "1.9.1";
-in mkDerivation {
+mkDerivation {
   pname = "qtcurve";
-  inherit version;
+  version = "1.9.1";
```

```suggestion
  version = "2020-06-21";
```
For commit https://github.com/KDE/qtcurve/commit/e534cd04e6acc90c105b492f3fcf99cc6155ba15